packages/html-reporter/bundle.ts

```diff
@@ -31,12 +31,15 @@ export function bundle(): Plugin {
           return html;
         html = html.replace(/(?=<!--)([\s\S]*?)-->/, '');
         for (const [name, value] of Object.entries(ctx.bundle) as any) {
+          const ext = path.extname(name);
           if (name.endsWith('.map'))
             continue;
-          if (value.code)
+          if (value.code && ext === '.js')
             html = html.replace(/<script type="module".*<\/script>/, () => `<script type="module">${value.code}</script>`);
-          else
+          else if (value.source && ext === '.css')
             html = html.replace(/<link rel="stylesheet"[^>]*>/, () => `<style type='text/css'>${value.source}</style>`);
+          else if (value.source && ext === '.svg' && name.includes('playwright-logo'))
```

Comment:
Can we make vite inline this automatically? Looking at https://vitejs.dev/config/build-options.html#build-assetsinlinelimit.